assignments/tasks_test.go

```diff
@@ -43,6 +43,11 @@ func TestSynchronizeTasksWithIssues(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	// Deleting issues failed with:
```

Comment:
@JosteinLindhom I would like to also fix this one before merging this one... let me know if you have found any clues regarding this. I'll see if I can figure it out as well...